lib/hexpm/web/router.ex

```diff
@@ -9,6 +9,7 @@ defmodule Hexpm.Web.Router do
     plug :fetch_session
     plug :fetch_flash
     plug :protect_from_forgery
+    plug Hexpm.Web.Plugs.Attack
```

Comment:
hmm, maybe it's a bad idea to use this plug in the browser, because it does e.g. rate limiting which we haven't done before for web requests. But it was convenient to add the rule to plug attack (I'd expect all rules like that to be there). wdyt @ericmj?

Reply:
We can use `PlugAttack` but not in the same plug that does throttling because we shouldnt rate limit in the bbrowser like you said. Maybe make a separate plug with all rules except throttling.

Reply:
@ericmj I ended up writing a simple function plug that does this for now, see the other PR.

@michalmuskala do you have some ideas how to either compose (or conditionally use parts of) Plug.Attack?